bluebottle/payments/services.py

```diff
@@ -1,6 +1,6 @@
 from bluebottle.payments.exception import PaymentException
 import re
-from django.conf import settings
+from bluebottle.clients import properties
```

Comment:
I thought the idea was to be able to extend the django settings based on the client. Shouldn't be able to do getattr(settings, 'PAYMENT_METHODS', ()) so that applications which use settings will "just work".